### PR TITLE
[es] Localize tightly-coupled-architectures.md

### PR DESCRIPTION
**Pull request by ai-naymul**
### Describe your changes
Localized the content/en/tightly-coupled-architectures.md document to Spanish 

### Related issue number or link (ex: `resolves #issue-number`)
Tracking issue:#1779

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`)
    is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco).
    If you are working locally, you could add an alias to your `gitconfig` by running
    `git config --global alias.ci "commit -s"`.

 Signed-off-by: Naymul Islam naymul504@gmail.com

### CODE REVIEW

**Comment by ai-naymul** [comment]
I am a newbie to contribute to open source It's my first contribtion so if there is any mistake that I have done please give me review of that
Thank you :)

**Comment by ramrodo** [comment]
Hi @ai-naymul, thanks for your contribution, and congrats on your first contribution! :)

Before we can merge your PR, all the tests should pass. Right now, the [DCO verification is failing](https://github.com/cncf/glossary/pull/1854/checks?check_run_id=12366217887).

To solve this, you need to sign off your commits. You can find more info on how to do/set up this in the template at opening a PR:

```
[ ] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`)
    is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco).
    If you are working locally, you could add an alias to your `gitconfig` by running
    `git config --global alias.ci "commit -s"`.
```

It's probably that you need to amend your commits or delete those and create them again now with signing off them.

File: content/es/tightly-coupled-architectures.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+título: Arquitecturas fuertemente acopladas
+Estado: COMPLETADO
+categoría: Propiedad
+etiquetas: ["fundamental", "arquitectura", "propiedad"]
```
Comment (ramrodo) [inline content/es/tightly-coupled-architectures.md:5]: This whole section should not be translated. See [this other Spanish localization](https://github.com/cncf/glossary/blob/dev-es/content/es/distributed-systems.md?plain=1#L2-L5) where we keep the words in the original language.

File: content/es/tightly-coupled-architectures.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+---
+
+## Lo que es
+
+La arquitectura estrechamente acoplada es un estilo arquitectónico en el que varios componentes de la aplicación son interdependientes.(el paradigma opuesto de [arquitecturas débilmente acopladas](/arquitectura débilmente acoplada/)).Esto significa que un cambio en un componente probablemente afectará a otros componentes. Por lo general, es más fácil de implementar que los estilos arquitectónicos acoplados de forma más flexible, pero puede dejar un sistema más vulnerable a fallas en cascada. También tienden a requerir implementaciones coordinadas de componentes que pueden volverse un lastre para la productividad del desarrollador.
```
Comment (ramrodo) [inline content/es/tightly-coupled-architectures.md:10]:
```suggestion
La arquitectura estrechamente acoplada es un estilo arquitectónico en el que varios componentes de la aplicación son interdependientes (el paradigma opuesto de [arquitecturas débilmente acopladas](/arquitectura débilmente acoplada/)).Esto significa que un cambio en un componente probablemente afectará a otros componentes. Por lo general, es más fácil de implementar que los estilos arquitectónicos acoplados de forma más flexible, pero puede dejar un sistema más vulnerable a fallas en cascada. También tienden a requerir implementaciones coordinadas de componentes que pueden volverse un lastre para la productividad del desarrollador.
```

**Review by ramrodo — CHANGES_REQUESTED** [review]
It seems that you're taking a deprecated version of the file. The most recent version of this concept is:

https://github.com/cncf/glossary/blob/main/content/en/tightly-coupled-architectures.md?plain=1

and this localization differs from the paragraphs you translated.

Please use the most recent version and submit your changes again.

**Review by ai-naymul — COMMENTED** [review]
I have changed the path and the header of the file
Thank you :)

**Comment by ai-naymul** [comment]
Hello ramrodo,
Please check the file is that correct or not or is there any mistake I have done let me know
and this is my first time that's why all of these happening :)

Sorry for your inconvenience

File: content/es/tightly-coupled-architectures.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+category: Property
+tags: ["fundamental", "architecture", "property"]
+---
+
+La arquitectura estrechamente acoplada es un estilo arquitectónico en el que varios componentes de la aplicación son interdependientes
+(el paradigma opuesto de [arquitecturas débilmente acopladas](/es/loosely-coupled-architecture/)).
+Esto significa que un cambio en un componente probablemente afectará a otros componentes.
+Por lo general, es más fácil de implementar que los estilos arquitectónicos más vagamente acoplados,
+pero puede dejar un sistema más vulnerable a fallas en cascada.
+También tienden a requerir lanzamientos coordinados de componentes.
```
Comment (ramrodo) [inline content/es/tightly-coupled-architectures.md:13]:
```suggestion
También tienden a requerir lanzamientos coordinados de componentes
```

File: content/es/tightly-coupled-architectures.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+lo que puede convertirse en un lastre para la productividad del desarrollador.
+
+Las arquitecturas de aplicaciones estrechamente acopladas son una forma bastante tradicional de crear aplicaciones.
+Si bien no es necesariamente consistente con todas las mejores prácticas de desarrollo de [microservicios](/microservicios/)
```
Comment (ramrodo) [inline content/es/tightly-coupled-architectures.md:17]:
```suggestion
Si bien no es necesariamente consistente con todas las mejores prácticas de desarrollo de [microservicios](/es/microservices-architecture/)
```

File: content/es/tightly-coupled-architectures.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+pueden ser útiles en circunstancias específicas.
+Tienden a ser más rápidos y sencillos de implementar y
+al igual que las [aplicaciones monolíticas](/monolithic-apps/), pueden acelerar el ciclo de desarrollo inicial.
```
Comment (ramrodo) [inline content/es/tightly-coupled-architectures.md:20]:
```suggestion
al igual que las [aplicaciones monolíticas](/es/monolithic-apps/), pueden acelerar el ciclo de desarrollo inicial.
```

**Comment by ai-naymul** [comment]
I have change those please check that correct or not or is there any mistake I have done let me know
And please give me some tips why my DCO is faillng again and again,I am commiting the "signed-of-by" everytime but it's failing please help me in that case
 #case?https://github.com/cncf/glossary/pull/1854/commits/053bb51c437c013817b7d6c0732f718ca1152d7a

**Comment by ai-naymul** [comment]
> > I have change those please check that correct or not or is there any mistake I have done let me know And please give me some tips why my DCO is faillng again and again,I am commiting the "signed-of-by" everytime but it's failing please help me in that case how can I solve this #case?[053bb51](https://github.com/cncf/glossary/commit/053bb51c437c013817b7d6c0732f718ca1152d7a)
> 
> @ai-naymul sure. In order to fix and sign-off all the previous commits, you could follow the instructions given in the [DCO test details](https://github.com/cncf/glossary/pull/1854/checks?check_run_id=12385104199):
> 
> image
> 
> Since in the beginning, many commits were missing the sign-off, you should do a `git ammend` or `git rebase` to re-write the commit messages.
> 
> Let me know if this information helps you.

 I have made a mistake by doing a empty commit to try to fix the failure,I try a lot ,Can I delete the whole pull request and make it again to fix that?

**Comment by 92nqb** [comment]
Hi @ai-naymul, 

have you tried with the command `git rebase HEAD~15 --signoff`?  I think that it would fix the issue with the DCO.

Let me know if it works :)

**Comment by ai-naymul** [comment]
> git rebase HEAD~15 --signoff

Hello @92nqb,
I try it and after trying the `tightly-coupled-architectures.md` file is deleted from the `content/es` directory .
So what should I do now, make the pull request again or not?
I am extremely sorry for you inconvenience 
Thank you :)

File: assets/scss/_cookie-banner.scss
Comment (krol3) [inline assets/scss/_cookie-banner.scss:1]: the Spanish translation only do changes in the `content/es`, for any doubt please join us to our calls also you can find groups in your local languages to collaborate @ai-naymul

**Review by krol3 — CHANGES_REQUESTED** [review]

**Comment by ai-naymul** [comment]
Hello @ramrodo ,
I have been facing problem with rebasing the commits,could you please help me?
should I use `git reset --soft HEAD~66` to reset all commit and make it again?
Thank you :)

**Comment by ramrodo** [comment]
> Hello @ramrodo , I have been facing problem with rebasing the commits,could you please help me? should I use `git reset --soft HEAD~66` to reset all commit and make it again? Thank you :)

Right @ai-naymul, resetting with `--soft` is a good option to re-write the commits and add the sign-off.

**Pull request closed without merging**